app/api/dao/admin.py

```diff
@@ -2,9 +2,27 @@
 
 
 class AdminDAO:
+    """ Class definition for AdminDAO.
 
+    Provides functions related to admins.
+
+    Functions:
```

Comment:
@isabelcosta since the style docs do not mention them I can remove that. Yes it has been declared in the same file but maybe including them would have given them a complete documentation. Overall we can ignore them.